@xen-orchestra/lite/src/views/vm/VmConsoleView.vue

```diff
@@ -1,7 +1,20 @@
 <template>
   <div v-if="!isReady">Loading...</div>
   <div v-else-if="!isVmRunning">Console is only available for running VMs.</div>
-  <RemoteConsole v-else-if="vmConsole" :location="vmConsole.location" />
+  <RemoteConsole
+    v-else-if="vmConsole"
+    :location="vmConsole.location"
+    :is-console-available="
+      !isOperationsPending(vm, [
+        'clean_shutdown',
+        'hard_shutdown',
+        'clean_reboot',
+        'hard_reboot',
+        'pause',
+        'suspend',
```

Comment:
Is it better to have a constant that contains this list to be verified? Perhaps it would be more maintainable.